Mlem/Views/Tabs/Feeds/Feed View.swift

```diff
@@ -93,7 +112,26 @@ struct FeedView: View {
             /// [2023.08] Set to `.visible` to workaround bug where navigation bar background may disappear on certain devices when device rotates.
             .navigationBarColor(visibility: .visible)
             .environmentObject(postTracker)
-            .task(priority: .userInitiated) { await initFeed() }
+            .task(priority: .userInitiated) {
+                if !isWaitingForSiteInformation {
+                    await initFeed()
+                }
+            }
+            .onChange(of: siteInformation.version) { newValue in
+                if let siteVersion = newValue {
```

Comment:
```suggestion
            .task(id: siteInformation.version) {
                if let siteVersion = siteInformation.version {
```

Making this a .task ensures that it performs the first check once the view is properly rendered, ensuring that a published update can't slip through the cracks